integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/support/TestCaseRunner.java

```
@@ -36,7 +36,7 @@ public TestCaseResult runWsdlTestCase(final TestCase testCase, final Map<String,
         final WsdlTestCaseRunner wsdlTestCaseRunner = new WsdlTestCaseRunner(wsdlTestCase, new PropertiesMap());
 
         try {
-            Thread.sleep(30000);
+            Thread.sleep(25000);
         } catch (final InterruptedException ex) {
             Thread.currentThread().interrupt();
```

Thread has already been interrupted, if desired you can log that interrupt (the exception here), normally you would just for example leave the method here, you can also just throw the exception.